Mathlib/CategoryTheory/Monoidal/Braided/Transport.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2025 Dagur Asgeirsson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dagur Asgeirsson
+-/
+import Mathlib.CategoryTheory.Monoidal.Braided.Basic
+import Mathlib.CategoryTheory.Monoidal.Transport
+
+/-!
+
+# Transport a symmetric monoidal structure along an equivalence of categories
+-/
+
+universe v₁ v₂ u₁ u₂
+
+open CategoryTheory Category Monoidal MonoidalCategory
+
+noncomputable section
```

Comment:
Should these be computable?

Reply:
The reason it's noncomputable is that it depends on `Functor.preimage`, which uses choice. We could maybe change `braidedCategoryOfFullyFaithful` to use `Functor.FullyFaithful`, which contains data, to make it computable, but I think it's not worth it, it's nicer to be able to use typeclass inference by using `Functor.Full` and `Functor.Faithful`.